packages/atomic/src/components/atomic-in-stencil/atomic-in-stencil.tsx

```diff
@@ -0,0 +1,87 @@
+import {Component, Element, h, State} from '@stencil/core';
+import {DeferUntilRender} from '../../utils/initialization-utils';
+
+const template = '<h1>LINK<h1><atomic-result-link></atomic-result-link>';
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'atomic-in-stencil',
+  shadow: true,
+})
+export class AtomicInStencil {
```

Comment:
I'm not sure I understand perfectly what the goal is of this component ?

I see it's marked as internal, so it's for "testing" ? 

And also, the sample is put in `packages/atomic/src/pages/examples/stencil.html`. If we want to follow the logic of the rest of the samples/"output target", should we consider Stencil just like React or Angular or VueJS ?

So we could have a `packages/samples/stencil` package, where we showcase implementation of Atomic inside Stencil, with README, code samples, cypress smoke tests, etc.